lib/cdo/rack/attack.rb

```diff
@@ -0,0 +1,59 @@
+require 'rack/attack'
+require 'active_support/core_ext/numeric/time'
+
+class Rack::Attack
+  # maximum requests per minute over a 4-minute stretch.
+  MAX_TABLE_READS_PER_MIN = 2 * 60
```

Comment:
I believe DCDO is pretty easy.  Just do DCDO.get instead of the constants, and have a script or controller on production daemon that resets the limits.

The benefit is high because it means we can increase limits without redeploying, rather than having things appear broken for users.  We want to block broken apps, but we want "well-behaved" apps to work.  (e.g. imagine an app used by everyone in a class or school, where you could legitimately get tens or hundreds of reads or writes per second.)